### PR TITLE
 add CategoryRepository

### PR DESCRIPTION
**Pull request by myzonchuk**

### CODE REVIEW

File: src/main/java/com/softserve/ita/java442/cityDonut/repository/CategoryRepository.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.softserve.ita.java442.cityDonut.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CategoryRepository extends JpaRepository {
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/repository/CategoryRepository.java]: JpaRepository has Generic

File: src/main/java/com/softserve/ita/java442/cityDonut/repository/CategoryRepository.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package com.softserve.ita.java442.cityDonut.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
```
Comment (martaprotsak) [inline src/main/java/com/softserve/ita/java442/cityDonut/repository/CategoryRepository.java:6]: Add @Repositiry

File: src/main/java/com/softserve/ita/java442/cityDonut/repository/CategoryRepository.java (hunk 1 above)
Comment (martaprotsak) [inline src/main/java/com/softserve/ita/java442/cityDonut/repository/CategoryRepository.java]: Maybe, it should "extends JpaRepository<Category, Long>"

Comment (nastiamazurak) [reply src/main/java/com/softserve/ita/java442/cityDonut/repository/CategoryRepository.java]: Definitely, JpaRepository is Generic and should extend <Category, Long>

File: src/main/java/com/softserve/ita/java442/cityDonut/repository/CategoryRepository.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package com.softserve.ita.java442.cityDonut.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/repository/CategoryRepository.java:7]: Where is import Category?

**Review by NazarRyvak — APPROVED** [review]

**Review by martavoloshyn — APPROVED** [review]

**Pull request merged**